types/ramda/ramda-tests.ts

```diff
@@ -2249,6 +2249,10 @@ class Rectangle {
     defaultTo42(null);  // => 42
     defaultTo42(undefined);  // => 42
     defaultTo42("Ramda");  // => 'Ramda'
+
+    const maybeValue = 2 as number | undefined;
```

Comment:
WDYT about `const maybeValue: number | undefined = 2`

Reply:
I've tried that at first, but it seemed that compiler just drops the `undefined` and infers the `maybeValue` as a `number`. Using `as` preserves the ` | undefined` part of the type.